src/Helpers/helperSlugify.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+if (!function_exists('isBreadSlugAutoGenerator')) {
+    /**
+     * Check if a slug field can be auto generated.
+     *
+     * @param json $options
+     *
+     * @return string HTML output.
+     */
+    function isBreadSlugAutoGenerator($options)
+    {
+        if (isset($options->slugify)) {
+            return ' data-slug-origin='.$options->slugify->origin
+                   .((isset($options->slugify->forceUpdate))
+                      ? ' data-slug-forceupdate=true'
+                      : '');
+        }
+    }
+}
+
+if (!function_exists('isFieldSlugAutoGenerator')) {
+    /**
+     * Determine the details field, for a given dataTypeContent.
+     *
+     * @param Illuminate\Database\Eloquent\Collection $dataTypeContent
+     * @param string                                  $field
+     *
+     * @return string HTML output.
+     */
+    function isFieldSlugAutoGenerator($dataTypeContent, $field)
+    {
+        $_tmp = (isset($dataTypeContent->id))
```

Comment:
I pushed a change into your other pull request that handled exceptions.

Add `$this->disableExceptionHandling();` to the start of `RouteTest`.
That change should be committed into `master` for better understanding of the route failures.